plutarch-test/plutarch-extra/Plutarch/ExtraSpec.hs

```diff
@@ -0,0 +1,8 @@
+module Plutarch.ExtraSpec (spec) where
+
+import Test.Syd (Spec, describe, it)
+
+spec :: Spec
+spec =
+  describe "Hello Plutarch Extra" $
```

Comment:
Note that by convention (when using golden tests) we do not use whitespace in `describe` strings.